src/features/calendar/components/EventShiftModal/index.tsx

```diff
@@ -26,9 +26,7 @@ const EventShiftModal: FC<EventShiftModalProps> = ({ close, dates, open }) => {
   const { orgId, campId } = useNumericRouteParams();
 
   const [type, setType] = useState<ZetkinEvent['activity']>(null);
-  const [eventTitle, setEventTitle] = useState<string>(
-    messages.eventShiftModal.noTitle()
-  );
+  const [eventTitle, setEventTitle] = useState<string>('');
```

Comment:
This is empty string, which means that an empty string is being sent to the server when this event is created. I think it would be better if the data being sent to the server was `null`, or that the `title` field wasn't included at all in the payload being sent to the server. You could keep the empty string default here, and then elsewhere (where the payload is being assembled for the server) if the string is empty, replace it with `null`.

Reply:
Yes sounds better with `null` ! will fix it right away!

Reply:
@richardolsson When I create a single event (by dragging on a calendar), it has an empty string ("") as the default value for the title. Should we also consider changing the default title value of single event to null?

Reply:
Yeah, I think that's a good idea. Will you create a small entry-level issue for this?

Reply:
Yes! Will do :)